spec/i18n_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'i18n/tasks'
+
+RSpec.describe I18n do
+  let(:i18n) { I18n::Tasks::BaseTask.new }
+  let(:missing_keys) { i18n.missing_keys }
+  let(:unused_keys) { i18n.unused_keys }
+  let(:inconsistent_interpolations) { i18n.inconsistent_interpolations }
+
+  it 'does not have missing keys' do
+    skip 'Enable when i18n-tasks is enabled across the project, otherwise this spec will report false positives'
```

Comment:
I always prefer `pending` with these kinds of to-do specs. We'll know it's fixed once it starts reporting as a failure

Reply:
Good call 👍🏾